.changelog/532.bugfix.2.md

```diff
@@ -0,0 +1 @@
+Increase md breakpoint value
```

Comment:
Why do we have two changelog fragments, instead one, with two lines? (Is this how it's supposed to work?)

Reply:
I am following one of the comment I've received in the past 
https://github.com/oasisprotocol/oasis-core-ledger/pull/261#discussion_r1174122546